lib.ps1

```diff
@@ -182,7 +182,11 @@ Function doFetchRemoteBranches {
 	Push-Location $Path
 	
 	# Remove all tracked branches from all remotes
-	git branch -r | xargs -l1 git branch -rD
+	$branches = git branch -r

```

Comment:
Remove this duplicated line

Reply:
Solved in 5610d05724fc8b93540aeed173f28e892a4d3689